packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -143,8 +143,26 @@ abstract class Action<T extends Intent> with Diagnosticable {
   /// [Actions.invoke], or by invoking it using an [ActionDispatcher]. An action
   /// invoked via a [Shortcuts] widget will have its return value ignored.
   @protected
-  Object? invoke(covariant T intent);
+  Object? invoke(T intent);
 
+  /// Called when the action is invoked as the override of another [Action].
+  ///
+  /// An [Action] can "override" a different [Action] if they both handled the
+  /// same type of [Intent]. An overriding [Action] may wish to invoke the
+  /// [Action] it overrides at some point, similar to calling the super
+  /// implementation in a method override.
+  ///
+  /// The [fromAction] parameter represents the overridden [Action]. To invoke
+  /// the "super" implementation in this method, call `fromAction.invoke`.
+  ///
+  /// This method is typically invoked in the overridden action's [invoke] or
```

Comment:
Which method are you referring to when you say "This method"?  "fromAction.invoke", or "invokeAsOverride"?  I'd just be explicit.